src/Zynga/Framework/Environment/CodePath/V1/CodePathTest.hh

```diff
@@ -6,31 +6,36 @@ use Zynga\Framework\Environment\CodePath\V1\CodePath;
 use Zynga\Framework\Testing\TestCase\V2\Base as TestCase;
 
 class CodePathTest extends TestCase {
-  private string $_currentRoot = '';
+  private string $currentRoot = '';
+  private string $currentExternalLibraryPath = '';
 
-  public function captureCurrentRoot(): void {
-    $this->_currentRoot = CodePath::getRoot();
+  public function setUp(): void {
+    parent::setUp();
+    $this->currentRoot = CodePath::getRoot();
+    $this->currentExternalLibraryPath = CodePath::getExternalLibraryPath();
   }
 
-  public function restoreCurrentRoot(): void {
-    CodePath::setRoot($this->_currentRoot);
+  public function tearDown(): void {
+    parent::tearDown();
```

Comment:
minor: usually safer to call parent::tearDown at the end instead of the beginning.

Reply:
Sure. Will change!